app/assets/stylesheets/modern-business.scss

```diff
@@ -96,11 +96,19 @@ header.carousel .fill#slider-3 {
   padding-top: 60px
 }
 
-#header-into {
+#header-intro {
   text-align: center;
   font-size: 18px
 }
 
+.donation-candidate h3:hover {
+  color: red
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Declaration should be terminated by a semicolon